server/controllers/sharing.coffee

```diff
@@ -12,14 +12,20 @@ mailTemplate = notiftemplate = localization.getEmailTemplate 'sharemail.jade'
 
 clearanceCtl = clearance.controller
     mailTemplate: (options, callback) ->
-        options.type = options.doc.docType.toLowerCase()
-        User.getDisplayName (err, displayName) ->
-            options.displayName = displayName or \
-                                  localization.t 'default user name'
-
+        # Use async to retrieve all wanted informations
+        async.parallel [
```

Comment:
You're true, but we also use the `getDisplayName` method elsewhere, and I don't want to broke the current code. Do we make another method to get both name and email values instead?